packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -289,7 +289,7 @@ class _DevFSHttpWriter {
     while ((_inFlight < kMaxInFlight) && (!_completer.isCompleted) && _outstanding.isNotEmpty) {
       final Uri deviceUri = _outstanding.keys.first;
       final DevFSContent content = _outstanding.remove(deviceUri);
-      _startWrite(deviceUri, content);
+      _startWrite(deviceUri, content, /* retry= */ 10);
```

Comment:
I would update this to a named argument instead of using the comment syntax.

```suggestion
      _startWrite(deviceUri, content, retry: 10);
```

Reply:
Done